### DISCUSSION TITLE
Fastlogin doesn't work with Authme Reloaded on 1.19

### DISCUSSION THREAD

**Discussion by MadPanCake** [games647/FastLogin#888] [Q&A]
Hello,

I'm running a 1.19 server
**Paper engine:** git-Paper-95
**Authme Reloaded:** v5.6.0-SNAPSHOT b2576, which is the latest
**FastLogin:** one of the "development" version, because I don't know how to download the official release, except of the Spigot one, which is from year 2017.

![image](https://user-images.githubusercontent.com/34662922/184610821-c11eb395-25ad-4f70-903c-4da576790702.png)

When someone joins I get his error:
![image](https://user-images.githubusercontent.com/34662922/184611571-67c769b0-4cb2-4357-adb7-38bb51290513.png)
And the auto-login for premium player is not working, even though I did invoke /premium twice.
It keeps asking for password.

The FastLogin has an asterisk after it's name in /plugins:
![image](https://user-images.githubusercontent.com/34662922/184611735-52cf9ee6-945a-42af-801b-c1b157951ba8.png)

**Comment by TuxCoding** [comment]
What is the version of FastLogin you are using?

You can use `version FastLogin` or look at the startup section of your server log. The error you posted indicates that the version you are using is not compatible with 1.19.

BTW: This area is more about questions not about errors/bugs. There is a template for all the data we need in the bug report area.

**Reply by MadPanCake** [reply]
![image](https://user-images.githubusercontent.com/34662922/184647236-b83a723e-94fa-4667-9253-d1d0a33b836c.png)
Well, I don't know how to download the latest version which will work for 1.19
It's not because I'm lazy, I geniuinely don't know how to download it from Github, and Spigotmc has the 1.12 version.

**Reply by TuxCoding** [reply]
Check for development builds.

**Reply by MadPanCake** [reply]
I downloaded newest version and it now works. Thank You for help :)
